webclient/components/1time-Mrow-mField/diagnosis/db/client-side/structure/master-table-of-diagnosis.js

```diff
@@ -18,8 +18,8 @@ export default class clientSideTblMasterDiagnosis extends clientSideTableManage
 
       masterDiagnosisId: this.uid(() => intUniqueId()),
       diagnosisName: this.string(null),
+      isAssignedToPatient: this.boolean(false),
```

Comment:
Why is this needed?

Reply:
Sir, 'isAssignedToPatient' field use for when the diagnosis assigned to a patient then the diagnosis remove from the master list of diagnosis.